src/App.tsx

```diff
@@ -11,6 +9,13 @@ import i18n from './i18n';
 import { useTranslation } from 'react-i18next';
 
 import Button from '@mui/material/Button';
+import { AppBar, Toolbar, MenuItem } from '@mui/material';
+import Typography from '@mui/material/Typography';
```

Comment:
I think it's good to be consistent, either import all components as default exports, or use named exports everywhere :) 
```
import AppBar from '@mui/material/AppBar';
import Toolbar from '@mui/material/Toolbar';
import MenuItem from '@mui/material/MenuItem';


```

Reply:
changed